WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -363,12 +364,12 @@ private void setComment(int localBlogId, final Comment comment) {
             showComment();
     }
 
-    private void setShouldFocusReplyField(boolean shouldFocusReplyField) {
-        mShouldFocusReplyField = shouldFocusReplyField;
+    private void setShouldFocusReplyField() {
```

Comment:
Both `setShouldFocusReplyField` and `setShouldRequestCommentFromNote` are a bit odd. I think it's fine to have them set the variable to true/false without passing a parameter, but their names should reflect what they're doing (ex: `disableShouldFocusReplyField`). Or just leave them the way they were.